docs/book/getting-started/zenml-pro/saas-deployment.md

```diff
@@ -0,0 +1,196 @@
+- Azure Key Vault
+- HashiCorp Vault
+
+![SaaS with customer secret store](../../.gitbook/assets/cloud_architecture_saas_detailed_2.png)
+
+This keeps all credentials within your infrastructure while still benefiting from managed ZenML services - [Book a call](https://www.zenml.io/book-your-demo) with us if you want this set up.
+
+## Network Architecture
+
+### Outbound-Only Communication
+
+ZenML Pro SaaS uses outbound-only connections from your infrastructure to ZenML services:
+- No inbound connections required to your infrastructure
+- Compatible with firewall and VPN restrictions
+- Secure WebSocket connections for real-time updates
+
+### Artifact Store Access
+
+The ZenML dashboard requires read access to your artifact store to display:
+- Pipeline visualizations
+- Model comparison views
+- Artifact lineage graphs
+- Step logs and outputs
+
+You control this access by configuring appropriate cloud IAM permissions.
+
+## Getting Started
+
+### 1. Sign Up
+
+[Book a demo](https://www.zenml.io/book-your-demo) to get started with ZenML Pro SaaS.
+
+### 2. Connect Your Cloud
+
+Configure access to your cloud infrastructure:
+- Set up an artifact store (S3, GCS, Azure Blob, etc.)
+- Configure compute resources (AWS, GCP, Azure, or Kubernetes)
+- Provide necessary credentials via secrets
+
+### 3. You're ready to run your pipelines and monitor them through the Frontend
+
+## Security Documentation
+
+For software deployed on your infrastructure, ZenML provides:
+
+- **Vulnerability Assessment Reports**: Comprehensive security analysis available on request
+- **Software Bill of Materials (SBOM)**: Complete dependency inventory for compliance
+- **Compliance documentation**: Support for your security audits and certifications
+
+Contact [cloud@zenml.io](mailto:cloud@zenml.io) to request security documentation.
+
+## Pricing & Support
+
+ZenML Pro SaaS includes:
+- Managed infrastructure and updates
+- Professional support with SLA
+- Regular security patches and updates
+- Access to pro-exclusive features
+- Usage-based pricing model
+
+[Contact us](https://www.zenml.io/book-your-demo) for pricing details and custom plans.
+
+## Comparison with Other Deployments
+
+| Feature | SaaS | Hybrid SaaS | Self-hosted |
+|---------|------|-------------|------------|
+| Setup Time | ⚡ Minutes | Hours | Days |
+| Maintenance | Zero | Workspace only | Full stack |
+| Infrastructure Control | Minimal | Moderate | Complete |
+| Data Sovereignty | Metadata on ZenML | Full | Full |
+| Best For | Fast time-to-value | Security requirements | Strictest compliance |
+
+[Compare all deployment options →](README.md#deployment-scenarios)
+
+## Migration Path
+
+Already running ZenML OSS? Migrating to SaaS is straightforward:
+
+1. **Export your data**: Use ZenML's migration tools
+2. **Set up SaaS workspace**: Configure your cloud connections
+3. **Import metadata**: Transfer existing pipeline history
+4. **Update pipelines**: Point to your new ZenML server
```

Comment:
This entire section feels out of place. You only have to mention that it's possible to migrate an OSS server to SaaS/Pro while keeping the existing data and URL. The only thing that would have to change is how users are authenticating.